Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Dockerfile for cBioPortal
+FROM elementolab/jamm:1.0
+MAINTAINER Alexandros Sigaras <als2076@med.cornell.edu>
+#===========================#
+# Clone cBioPortal			#
+#===========================#
+#RUN git clone https://github.com/cBioPortal/cbioportal.git; \
+ADD / /cbioportal
+RUN	echo "export PORTAL_HOME=/cbioportal" >> /root/.bash_profile;
```

Comment:
Currently this loads from .bashrc.
The `ENV` command is the correct approach for variables configured during runtime.
Would configuring PORTAL_HOME, which is practically our project dir add any value or should we simply define a path in the dockerfile?

Personally leaning towards the latter but open to suggestions.